client/blocks/signup-form/index.jsx

```diff
@@ -338,19 +362,10 @@ class SignupForm extends Component {
 
 	handleBlur = ( event ) => {
 		const fieldId = event.target.id;
-		// Ensure that username and password field validation does not trigger prematurely
-		if ( fieldId === 'password' ) {
-			this.setState( { focusPassword: true }, () => {
-				this.validateAndSaveForm();
-			} );
-			return;
-		}
-		if ( fieldId === 'username' ) {
-			this.setState( { focusUsername: true }, () => {
-				this.validateAndSaveForm();
-			} );
-			return;
-		}
```

Comment:
Were these lines doing nothing? My concern is that we might be changing the behavior of the app for the rest of the users even if the feature flag is disabled.

Reply:
Yes, I can't see that these lines were doing anything.  They set the `focusPassword` or `focusUsername` state properties before validating the form, but nothing else in the component references these state properties, so I don't see that they were doing anything at all. 🤷‍♂️

Reply:
Nice, thanks for the explanation!